app/src/main/java/com/teamzzong/hacker/presentation/battle/rank/BattleRankAdapter.kt

```diff
@@ -59,10 +62,15 @@ class BattleRankAdapter : RecyclerView.Adapter<CreateRankViewHolder>() {
                 tvTeamName.text = item.name
                 tvCommit.text = "${dec.format(item.commitCount)} 커밋"
             }
+            binding.root.setOnClickListener {
+                itemClick(item)
+            }
         }
     }
 
-    class RankViewHolder(private val binding: ItemRankBinding) :
+    class RankViewHolder(
+        private val binding: ItemRankBinding,
+        private val itemClick: (Teams) -> (Unit)) :
```

Comment:
```suggestion
        private val itemClick: (Teams) -> (Unit)
    ) : CreateRankViewHolder(binding) {
```

괄호 두개 겹쳐있을 땐 하나를 떼는 미덕...

Reply:
제가 정신이 없네요